apps/nextjs-website/src/components/organisms/Auth/ConfirmLogin.tsx

```diff
@@ -57,22 +55,22 @@ const ConfirmLogin = ({ email, onConfirmLogin }: confirmLoginProps) => {
               <IllusEmailValidation />
             </Stack>
             <Typography variant='h4' pt={8} mb={5} textAlign='center'>
-              {confirmLogin.title}
+              {confirmLogin('title')}
             </Typography>
             {email && (
               <Typography
                 variant='body2'
                 mb={6}
                 dangerouslySetInnerHTML={{
-                  __html: confirmLogin.body(email),
+                  __html: confirmLogin.raw('body').replace('{email}', email),
```

Comment:
I think is here is better to use [parametric translation](https://next-intl-docs.vercel.app/docs/usage/messages) of `next-intl` instead of replace

Reply:
We can't use params if we need custom HTML inside the translations